packages/vscode-extension/src/devices/DeviceBase.ts

```diff
@@ -244,4 +248,12 @@ export abstract class DeviceBase implements Disposable {
     }
     return this.previewStartPromise;
   }
+
+  public async startMaestroTest(fileNames: string[]) {
+    await this.runMaestroTest(fileNames);
+  }
+
+  public async stopMaestroTest() {
+    await this.abortMaestroTest();
+  }
```

Comment:
I don't like that we're adding more functionality to this class. It's already pretty bloated (imo), and launching `maestro` doesn't seem like something that the abstraction of a "Device" should be doing anyway.
Instead, I guess I'd like to see a `MaestroTestRunner` (or something) class, which:
- has the `start`/`stopMaestroTest` methods.
- updates the `MaestroTestState` state.
- in the constructor, gets passed the device id and other parameters needed to launch the test
- is probably initialized and managed by `DeviceSession`, since you wouldn't want to run the test on a non-running device (presumably)